### ISSUE TITLE
You can't change to the country view when the map is zoomed in

### ISSUE THREAD

**Issue by karindr** [hbz/oerworldmap#1434]
At the moment when zooming in when the heatmap changes to the map view with streets and so on you are no longer able to change to the country view by clicking on the map.

**Comment by literarymachine** [comment]
> At the moment when zooming in when the heatmap changes to the map view with streets and so on you are no longer able to change to the country view by clicking on the map.

This is expected behavior; on street level, you can't really see countries so it makes little sense to be able to switch them.

**Comment by karindr** [comment]
I don't agree. If I am on the map in Cologne I know that I am in Germany and when I want to change to the German country view I have to zoom out several times before I can get there or habe to got to the home button and then zoom in several times to get there too. This does not make sense to me.

**Comment by trugwaldsaenger** [comment]
I think I never tried to activate the country view from the street view level. I think it`s not so urgent - maybe something to keep in mind, when we do some user testing...

**Comment by formimkontext** [comment]
besides the fact that i don't think this is a major problem, too, we could implement an button (close to the legend) to set the map back to a certain zoom level. @dobladov allready dealed with it, right?

**Comment by karindr** [comment]
There is no major problem, yes. But there is also no reason why it depends on the zoom level that you are able to change to country view. Can you explain where is the problem is leading to the country view from every zoom level?

**Comment by literarymachine** [comment]
> Can you explain where is the problem is leading to the country view from every zoom level?

While testing the map, we often accidentally clicked slightly beside a pin on street level. It was very confusing to then suddenly jump and zoom into a country view.

**Comment by karindr** [comment]
Was this allready with the new points for the pins? At the moment it is quite clear that you have to click on the point when it is orange and enlaged and the label is shown, isn't it?
